core/src/main/java/jenkins/widgets/ExecutorsWidget.java

```diff
@@ -13,6 +21,78 @@
  * @author Kohsuke Kawaguchi
  * @since 1.514
  */
-@Extension(ordinal = 100) @Symbol("executors") // historically this was above normal widgets and below BuildQueueWidget
 public class ExecutorsWidget extends Widget {
+    private final String ownerUrl;
+    private final List<Computer> computers;
+
+    public ExecutorsWidget(@NonNull String ownerUrl, @NonNull List<Computer> computers) {
+        this.ownerUrl = ownerUrl;
+        this.computers = new ArrayList<>(computers);
+    }
+
+    @Override
+    protected String getOwnerUrl() {
+        return ownerUrl;
+    }
+
+    public List<Computer> getComputers() {
+        return Collections.unmodifiableList(computers);
+    }
+
+    @Extension(ordinal = 100) @Symbol("executors") // historically this was above normal widgets and below BuildQueueWidget
+    public static final class ViewFactoryImpl extends WidgetFactory<View, ExecutorsWidget> {
+        @Override
+        public Class<View> type() {
+            return View.class;
+        }
+
+        @Override
+        public Class<ExecutorsWidget> widgetType() {
+            return ExecutorsWidget.class;
+        }
+
+        @NonNull
+        @Override
+        public Collection<ExecutorsWidget> createFor(@NonNull View target) {
+            return List.of(new ExecutorsWidget(target.getUrl(), target.getComputers()));
+        }
+    }
+
+    @Extension(ordinal = 100) @Symbol("executors") // historically this was above normal widgets and below BuildQueueWidget
+    public static final class ComputerFactoryImpl extends WidgetFactory<Computer, ExecutorsWidget> {
+        @Override
+        public Class<Computer> type() {
+            return Computer.class;
+        }
+
+        @Override
+        public Class<ExecutorsWidget> widgetType() {
+            return ExecutorsWidget.class;
+        }
+
+        @NonNull
+        @Override
+        public Collection<ExecutorsWidget> createFor(@NonNull Computer target) {
+            return List.of(new ExecutorsWidget(target.getUrl(), List.of(target)));
+        }
+    }
+
+    @Extension(ordinal = 100) @Symbol("executors") // historically this was above normal widgets and below BuildQueueWidget
```

Comment:
Need different symbols I think.

Reply:
I don't think it really matters. A symbol must be unique within its extension point, but that's it.

Reply:
> A symbol must be unique within its extension point

Which in this case I would think is `WidgetFactory`, and you have three named `executors`.

Not sure there is any practical impact for now, since this is not something you can configure via JCasC. If https://plugins.jenkins.io/extension-filter/ were refreshed and made to support JCasC then it could be visible.